src/opm/parser/eclipse/EclipseState/Grid/MULTREGTScanner.cpp

```diff
@@ -109,6 +109,9 @@ namespace Opm {
             if (e3DProps.hasDeckIntGridProperty( record->region_name)) {
                 int srcRegion    = record->src_value;
                 int targetRegion = record->target_value;
+                if (srcRegion > targetRegion)
```

Comment:
Instead of this swap I would like the symmetrization to be performed by inserting both pair explcitly; i.e. something like:
```C++
if (srcRegion != targetRegion) {
   std::pair<int,int> pair1(srcRegion,targetRegion);
   std::pair<int,int> pars(targetRegion, srcRegion);

   searchPairs[pair1] = &(*record);
   searchPairs[pair2] = &(*record);
}
```
that should work better with the lookup method - which *is* direction dependent. Hopefully all tests should pass without any further updates then.

Reply:
Done!